setup.py

```diff
@@ -123,6 +123,49 @@ def get_cuda_path():
     return _cuda_path
 
 
+def get_sycl_path():
+    basekit = os.environ.get("CMPLR_ROOT")
+    if basekit:
+        return basekit
+    else:
+        # If basekit env is not set then try default path
+        return "/opt/intel/oneapi/compiler/latest/"
+
+
+def get_sycl_info():
+    sycl_info = {"compile": [], "libdirs": [], "libs": []}
+    sycl_path = get_sycl_path()
+    if not sycl_path:
+        return sycl_info
+
+    include_suffixes = [
+        "linux/include/",
+        "linux/include/sycl",
```

Comment:
Is OSX supported?

Reply:
No. Intel oneAPI Toolkits will not support macOS since the 2024.0 release.